osgar/lib/test_line.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+import math
+
+from .line import Line, pointAtPolyLineDist
```

Comment:
I don't remember - are we using absolute or relative imports? Or are we using a mix?